lstchain/datachecks/dl1_checker.py

```diff
@@ -29,31 +24,23 @@
 
 from astropy import units as u
 from astropy.table import Table, vstack
-from bokeh.io import output_file, show
-from bokeh.layouts import gridplot, column
-from bokeh.models import HoverTool, Div
-from bokeh.models.annotations import Title
-from bokeh.models.widgets import Tabs, Panel
-from bokeh.plotting import figure
 from ctapipe.coordinates import EngineeringCameraFrame
 from ctapipe.instrument import CameraGeometry
 from ctapipe.io import HDF5TableWriter
 from ctapipe.visualization import CameraDisplay
-from ctapipe.visualization.bokeh import CameraDisplay as BokehCameraDisplay
 from datetime import datetime
 from lstchain.datachecks.containers import DL1DataCheckContainer
 from lstchain.datachecks.containers import DL1DataCheckHistogramBins
 from lstchain.io.io import dl1_params_lstcam_key
 from lstchain.paths import parse_datacheck_dl1_filename, parse_dl1_filename, \
     run_to_muon_filename, run_to_datacheck_dl1_filename
+# from lstchain.visualization.bokeh import plot_mean_and_stddev_bokeh
```

Comment:
can this be simply removed?

Reply:
This is for the future bokeh version of the current run data check pdf. I prefer to keep it.

Reply:
I included here a workaround to solve https://github.com/cta-observatory/cta-lstchain/issues/518. 
I know, that fix does not really belong here, apologies for that, but we have to move on and I do not have time to make a separate PR and test it (would surely make a mess with my branches).